setup.py

```diff
@@ -50,7 +50,7 @@
 def _build_webapp():
     already_exists = Path(f"./taipy/gui/webapp/index.html").exists()
     if not already_exists:
-        os.system("cd gui && npm ci && npm run build")
+        os.system("cd gui && npm ci --omit=optional && npm run build")
```

Comment:
Shouldn't there be a --only=prod at this point?

Reply:
--only=prod is not supported since at least npm 6.3